### DISCUSSION TITLE
大佬们，请教一个问题，paddleocr是不是对图片大小以及行数，文字大小有要求阿

### DISCUSSION THREAD

**Discussion by jingmingtao** [PaddlePaddle/PaddleOCR#12600] [Q&A]
大佬们，请教一个问题，paddleocr是不是对图片大小以及行数，文字大小有要求阿。

**Comment by jingmingtao** [comment]
太少的字，太少的行，太大的字，太小的图就不行阿，我崩溃了阿，谁知道阿大佬们，求求你们了

**Comment by jingmingtao** [comment]
一直hmean为0，我调整了各种的config参数，各种config，各种与训练模型，训练的结果在20个epoch的时候指标还有，训练几个就会变为0.有时候直接就为0.咋回事阿，能不能百忙之中看一眼，ai大佬们

**Comment by jingmingtao** [comment]
我将图片外阔，刚开始，指标挺高然后也训练着就变为了0，咋回事阿

**Comment by Hegelim** [comment]
Calm down bro, this is not the end of the world. I'd suggest take a look of your training data. Does your training data cover those cases you mentioned? If not, I'd suggest add such cases to your training data.

**Comment by xireli** [comment]
ppocr-v2要求图片高度为32，ppocr-v3以上都是48，一张图片的字符数量不超过25个，其他没发现别的 有不足之处 各位可以补充🙏
